static/src/javascripts/projects/common/modules/commercial/acquisitions-link-enrichment.js

```diff
@@ -51,24 +55,39 @@ const addReferrerDataToAcquisitionLink = (rawUrl: string): string => {
 
 const ACQUISITION_LINK_CLASS = 'js-acquisition-link';
 
+const makeAcquisitionLinksRegionSpecific = (): Promise<void> =>
+    getGeolocation().then(countryCode => {
+        const supportInternationalisationId = countryToSupportInternationalisationId(
+            countryCode
+        );
+        const links = Array.from(
+            document.getElementsByClassName(ACQUISITION_LINK_CLASS)
+        );
+
+        links.forEach(el => {
+            const link = el.getAttribute('href');
+            if (link) {
+                el.setAttribute(
+                    'href',
+                    addCountryGroupToSupportLink(
+                        link,
+                        supportInternationalisationId
+                    )
+                );
+            }
+        });
+    });
+
 const addReferrerDataToAcquisitionLinksOnPage = (): void => {
     const links = Array.from(
         document.getElementsByClassName(ACQUISITION_LINK_CLASS)
     );
 
     links.forEach(el => {
-        fastdom
-            .read(() => el.getAttribute('href'))
-            .then(link => {
-                if (link) {
-                    fastdom.write(() => {
-                        el.setAttribute(
-                            'href',
-                            addReferrerDataToAcquisitionLink(link)
-                        );
-                    });
-                }
-            });
+        const link = el.getAttribute('href');
```

Comment:
I removed the fastdom stuff here. Neither getting nor setting the `href` on a link will cause layout thrashing since these are not actually visible to the user (expect by hovering on a link). So fastdom serves no purpose here, other than to complicate the logic.